src/components/Header/Header.tsx

```diff
@@ -0,0 +1,103 @@
+            />
+            <line
+              x1="7"
+              x2="14"
+              y1="25.1406"
+              y2="32.1406"
+              fill="none"
+              stroke="#000000"
+              strokeLinecap="round"
+              strokeLinejoin="round"
+              strokeMiterlimit="10"
+              strokeWidth="2"
+            />
+            <path
+              fill="none"
+              stroke="#B5460F"
+              strokeLinecap="round"
+              strokeLinejoin="round"
+              strokeMiterlimit="10"
+              strokeWidth="2"
+              d="M29.043,7.8457c0.1015,1.125,0.5605,2.2236,1.4209,3.084c1.9521,1.9512,1.9521,5.1172,0,7.0703 c-1.9522,1.9531-1.9522,5.1191,0,7.0703c1.9521,1.9512,1.9521,5.1172,0,7.0703c1.9521-1.9531,1.9521-5.1191,0-7.0703 c-1.9522-1.9512-1.9522-5.1172,0-7.0703c1.9521-1.9531,1.9521-5.1191,0-7.0703C29.6035,10.0693,29.1445,8.9707,29.043,7.8457"
+            />
+            <path
+              fill="none"
+              stroke="#B5460F"
+              strokeLinecap="round"
+              strokeLinejoin="round"
+              strokeMiterlimit="10"
+              strokeWidth="2"
+              d="M34.5703,7.0361C34.4697,8.4336,34.9316,9.8623,36,10.9297c1.9521,1.9512,1.9521,5.1172,0,7.0703 c-1.9521,1.9531-1.9521,5.1191,0,7.0703c1.9521,1.9512,1.9521,5.1172,0,7.0703c1.9521-1.9531,1.9521-5.1191,0-7.0703 c-1.9521-1.9512-1.9521-5.1172,0-7.0703c1.9521-1.9531,1.9521-5.1191,0-7.0703C34.9316,9.8623,34.4697,8.4336,34.5703,7.0361"
+            />
+            <path
+              fill="none"
+              stroke="#B5460F"
+              strokeLinecap="round"
+              strokeLinejoin="round"
+              strokeMiterlimit="10"
+              strokeWidth="2"
+              d="M40.0107,7.2832c-0.0302,1.3164,0.4483,2.6426,1.4532,3.6465c1.9521,1.9512,1.9521,5.1172,0,7.0703 c-1.9522,1.9531-1.9522,5.1191,0,7.0703c1.9521,1.9512,1.9521,5.1172,0,7.0703c1.9521-1.9531,1.9521-5.1191,0-7.0703 c-1.9522-1.9512-1.9522-5.1172,0-7.0703c1.9521-1.9531,1.9521-5.1191,0-7.0703C40.459,9.9258,39.9805,8.5996,40.0107,7.2832"
+            />
+          </g>
+        </svg>
+        <h1 className={styles['logo-title']}>Stonesoup</h1>
+      </div>
+      <nav className={styles.block}>
+        <NavLink to={URLS['/']} className={({ isActive }) => cn(isActive ? styles.active : styles.pending, styles.nav)}>
+          Home
+        </NavLink>
+        <NavLink
+          to={URLS.ingredients}
+          className={({ isActive }) => cn(isActive ? styles.active : styles.pending, styles.nav)}
+        >
+          Ingredients
+        </NavLink>
+        <NavLink
+          to={URLS.products}
+          className={({ isActive }) => cn(isActive ? styles.active : styles.pending, styles.nav)}
+        >
+          Products
+        </NavLink>
+      </nav>
+      <div className={styles.block}>
+        <AccountIcon color="accent" className={styles['icon-block']} />
```

Comment:
Везде надо поправить
```suggestion
        <AccountIcon color={ColorType.accent} className={styles['icon-block']} />
```

Reply:
done